exercises/practice/list-ops/src/lib.rs

```diff
@@ -0,0 +1,69 @@
+/// Yields each item of a and then each item of b
+pub fn append<T, I>(_a: I, _b: I) -> impl Iterator<Item = T>
+where
+    I: Iterator<Item = T>,
+{
+    // this empty iterator silences a compiler complaint that
+    // () doesn't implement Iterator
+    std::iter::empty()
```

Comment:
```suggestion
    std::iter::from_fn(|| todo!())
```

Let's use something like this as a placeholder iterator, so the "empty" tests don't pass at first.